core/declarations.gms

```diff
@@ -331,7 +331,7 @@ vm_emiCO2Sector(ttot,all_regi,emi_sectors)           "total CO2 emissions from i
 vm_emiTeDetail(ttot,all_regi,all_enty,all_enty,all_te,all_enty)  "energy-related emissions per region and technology"
 vm_emiTe(ttot,all_regi,all_enty)                     "total energy-related emissions of each region. [GtC, Mt CH4, Mt N]"
 vm_emiMacSector(ttot,all_regi,all_enty)              "total non-energy-related emission of each region. [GtC, Mt CH4, Mt N]"
-vm_emiCdr(ttot,all_regi,all_enty)                    "total (negative) emissions due to CDR technologies of each region. [GtC]"
+vm_emiCdr(ttot,all_regi,all_enty)                    "total (negative) emissions from CDR technologies of each region that are calculated in the CDR module. Note that it includes all captured emissions entering the CCUS chain (i.e. CO2 stored AND used) [GtC]"
```

Comment:
I propose to be more specific:
` Note that it includes atmospheric CO2 entering the CCUS chain, i.e. CO2 stored (CDR) AND used (no CDR)`
because it excludes captured CO2 from the DAC heating process (and potentially from the liming process of oae later on)

Reply:
Great suggestion, was added in all places :)